.scripts/prepublish.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const shell = require('shelljs');
+const chalk = require('chalk');
```

Comment:
Hi @UsulPro, we don't have chalk under `devDependncies`. If it's working, it must be because chalk is installed by some other module. Add the dependency or just do a simple echo. Does chalk work fine in windows?